File: programacao/python/respostas/operacoes/45.py
colesterol_total = float(input("Colesterol total (mg/dL): "))
hdl = float(input("HDL (mg/dL): "))
ldl = float(input("LDL (mg/dL): "))

triglicerideos = (colesterol_total - hdl - ldl) * 5
print(f"A estimativa de triglicerideos e: {triglicerideos:.2f} mg/dL")

File: programacao/c/respostas/operacoes/45.c
#include <stdio.h>

int main() {
    float colesterol_total, hdl, ldl;
    float triglicerideos;

    printf("Digite o colesterol total (mg/dL): ");
    scanf("%f", &colesterol_total);

    printf("Digite o LDL (mg/dL): ");
    scanf("%f", &ldl);

    printf("Digite o HDL (mg/dL): ");
    scanf("%f", &hdl);

    triglicerideos = (colesterol_total - hdl - ldl) * 5;

    printf("A estimativa de triglicerideos e: %.2f mg/dL\n", triglicerideos);

    return 0;
}
